packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -0,0 +1,279 @@
+import { BigNumber } from 'bignumber.js'
+import * as React from 'react'
+import { Clipboard, StyleSheet, View } from 'react-native'
+import css from 'src/dev/ValidatorsList.scss'
+import { H1 } from 'src/fonts/Fonts'
+import { I18nProps, withNamespaces } from 'src/i18n'
+import Chevron, { Direction } from 'src/icons/chevron'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
+import { colors, standardStyles, textStyles } from 'src/styles'
+
+interface ValidatorsListProps {
+  data: any
+  isLoading: boolean
+}
+
+export interface State {
+  expanded?: number
```

Comment:
typically i think of `?` as meaning the key could be missing. where as `expanded: number | undefined` says the key is there but value could be undefined